dataprofiler/profilers/graph_profiler.py

```diff
@@ -401,7 +402,15 @@ def _get_continuous_distribution(
 
                 for distribution in distribution_candidates:
                     # compute fit, mle, kolmogorov-smirnov test to test fit, and pdf
-                    fit = distribution.fit(df)
+
+                    # scipy 1.11.0 updated the way they handle
+                    # the loc parameter in fit() for lognorm
+                    scipy_version = float(scipy.__version__.replace(".", ""))
```

Comment:
to get the version, i suggest we do something like:
`pkg_resources.get_distribution('scipy').version`

I'm not 100% convinced that we should convert to float and compare as sometimes versions can contain strs.

Any other options avail?

Reply:
Actually, according to SO, `pkg_resources` is slow to import.
```python
# at top of file
import importlib.metadata

importlib.metadata.version('scipy')
```

Reply:
done!